City-of-Helsinki/drupal-helfi-platform
# UHF-9354 Node packages

# [UHF-9354](https://helsinkisolutionoffice.atlassian.net/browse/UHF-9354)


## What was done


* Updated node to 20.10.0.
* Updated eslint, eslint import resolver for webpack and eslint parser
* Updated babel core and babel preset env.

## How to install and test

* Checkout this branch `git checkout UHF-9354_node_packages`
* Go to `./public/themes/custom/hdbt_subtheme/` and run `nvm use && npm i && npm run build` and check that there is no errors or vulnerabilities

## Other PRs


* https://github.com/City-of-Helsinki/drupal-hdbt/pull/835
* https://github.com/City-of-Helsinki/drupal-hdbt-admin/pull/248
* https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/649
* https://github.com/City-of-Helsinki/drupal-helfi-etusivu/pull/478
* https://github.com/City-of-Helsinki/drupal-helfi-kuva/pull/252
* https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus/pull/498
* https://github.com/City-of-Helsinki/drupal-helfi-asuminen/pull/317
* https://github.com/City-of-Helsinki/drupal-helfi-sote/pull/703
* https://github.com/City-of-Helsinki/drupal-helfi-strategia/pull/537
* https://github.com/City-of-Helsinki/drupal-helfi-tyo-yrittaminen/pull/306
* https://github.com/City-of-Helsinki/drupal-helfi-kymp/pull/735
* https://github.com/City-of-Helsinki/drupal-helfi-rekry/pull/356



[UHF-9354]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-9354?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
